per/views.py

```diff
@@ -1,70 +1,34 @@
-import json, datetime, pytz
-from django.http import JsonResponse, HttpResponse
-from api.views import (
-    bad_request,
-    bad_http_request,
-    PublicJsonPostView,
-    PublicJsonRequestView,
-)
-from rest_framework import viewsets
-from django.contrib.auth.models import User
+from django.db import transaction
+from django.http import JsonResponse
+from django.utils import timezone
+from rest_framework import authentication, permissions
+from rest_framework.views import APIView
 from .models import (
     Draft, Form, FormData, WorkPlan, Overview
 )
-from rest_framework.authtoken.models import Token
-
-def create_draft(raw):
```

Comment:
@batpad @thenav56  Removed most of these unnecessarily abstracted functions. Most of these can be found now inside the Views, ex. this one:
https://github.com/IFRCGo/go-api/pull/840/commits/b8b8e9ec3ed34b28d0a90879096284facb702d64#diff-8e4a20b18006b7a31a84250b0828251eR182